fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/cameras.cfg

```diff
@@ -1,1662 +1,357 @@
-InitialPbh = -8, 0, 0
-NodesToHide =""
-ClipMode ="Normal"
-BoundingBoxRadius =0.1
-UITitle="TT:GAME.PANEL_CAMERA_PILOT_LANDING"
+Title="0"
+Guid="{755DA2FA-30B8-42FD-8B4F-796A284C62D7}"
+UITitle="View EFB"
+Description="Left EFB view of the captain"
+Origin="Virtual Cockpit"
+Track="None"
 TargetCategory="None"
+ClipMode="Normal"
+SnapPbhAdjust="None"
+PanPbhAdjust="None"
+XyzAdjust=0
+ShowAxis="NO"
+AllowZoom=1
+InitialZoom=0.18703
+SmoothZoomTime=5
+BoundingBoxRadius=0.1
+ShowWeather=0
 CycleHidden=0
 CycleHideRadius=0
 ShowPanel=0
+MomentumEffect=0
+ShowLensFlare=0
+PanPbhReturn=0
+SnapPbhReturn=1
 InstancedBased=0
 NoSortTitle=0
 Transition=0
-PanAcceleratorTime=5
-XYZRate=0.25
-XYZAcceleratorTime=0
-
+Category="Cockpit"
+SubCategory="Custom"
+SubCategoryItem="None"
+InitialXyz= -0.123601, 1.719999, 18.695095
+InitialPbh= -20.075787, -0.003193, -50.77211
 [CAMERADEFINITION.3]
-Title ="CoPilot"
-Guid ="{43E6820F-A5B3-4E40-B610-1BA5769C333F}"
-Description =""
-Origin ="Virtual Cockpit"
-MomentumEffect =1
-SnapPbhAdjust ="Swivel"
-SnapPbhReturn =0
-PanPbhAdjust ="Swivel"
-PanPbhReturn =0
-Track ="None"
-ShowAxis ="YES"
-AllowZoom =1
-InitialZoom =0.4
-SmoothZoomTime =2
-ZoomPanScalar =1
-ShowWeather =1
-XyzAdjust =1
-ShowLensFlare =0
-Category ="Cockpit"
-SubCategory ="Pilot"
-SubCategoryItem ="CoPilot"
-PitchPanRate =20
-HeadingPanRate =60
-InitialXyz = 1.05, 1.86, 18.6
-InitialPbh = -7.5, 0, 0
-NodesToHide =""
-ClipMode ="Normal"
-BoundingBoxRadius =0.1
-UITitle="TT:GAME.PANEL_CAMERA_PILOT_COPILOT"
+Title="3"
+Guid="{676FD75D-85BA-4144-9427-418E12BAF944}"
+UITitle="View MFD SD MFD"
+Description="View of the instruments"
+Origin="Virtual Cockpit"
+Track="None"
 TargetCategory="None"
+ClipMode="Normal"
+SnapPbhAdjust="None"
+PanPbhAdjust="None"
+XyzAdjust=0
+ShowAxis="NO"
+AllowZoom=1
+InitialZoom=0.150682
+SmoothZoomTime=5
+BoundingBoxRadius=0.1
+ShowWeather=0
 CycleHidden=0
 CycleHideRadius=0
 ShowPanel=0
+MomentumEffect=0
+ShowLensFlare=0
+PanPbhReturn=0
+SnapPbhReturn=1
 InstancedBased=0
 NoSortTitle=0
 Transition=0
-PanAcceleratorTime=5
-XYZRate=0.25
-XYZAcceleratorTime=0
-
+Category="Cockpit"
+SubCategory="Custom"
+SubCategoryItem="None"
+InitialXyz= 0.516986, 1.897541, 18.618258
+InitialPbh= -49.193199, 0.000062, -0.017444
 [CAMERADEFINITION.4]
-Title ="PFD"
-Guid ="{AA3050EA-D745-4FA7-817F-82E984CFF5FF}"
-Description =""
-Origin ="Virtual Cockpit"
-MomentumEffect =1
-SnapPbhAdjust ="Swivel"
-SnapPbhReturn =0
-PanPbhAdjust ="Swivel"
-PanPbhReturn =0
-Track ="None"
-ShowAxis ="YES"
-AllowZoom =1
-InitialZoom =0.2
-SmoothZoomTime =2
-ZoomPanScalar =1
-ShowWeather =1
-XyzAdjust =1
-ShowLensFlare =0
-Category ="Cockpit"
-SubCategory ="Instrument"
-SubCategoryItem ="None"
-PitchPanRate =20
-HeadingPanRate =60
-InitialXyz = 0.05, 1.6, 19
-InitialPbh = -18, 0, 0
-NodesToHide =""
-ClipMode ="Normal"
-BoundingBoxRadius =0.1
-UITitle="PFD/ND"
+Title="5"
+Guid="{F8DA40F6-7E12-484D-99E6-BE4DCE962883}"
+UITitle="Passenger View Window Flaps"
```

Comment:
We probably still want to use localization strings for this, just maybe our own instead of what is provided from the sim.

Reply:
They can use it; I think it's better for us to use our own strings.

Reply:
We do have our own loc pac for the A380 and the localazy system so they can get translated to the various languages https://github.com/flybywiresim/aircraft/tree/master/fbw-a380x/src/localization/msfs